index.html

```diff
@@ -20,10 +20,139 @@
+                            <td>When</td>
+                            <td>What</td>
+                            <td>Where</td>
+                        </tr>
+                    </thead>
+                    <tr>
+                        <td>5:00pm</td>
+                        <td>Check-in opens</td>
+                        <td>CUC McConomy</td>
+                    </tr>
+                    <tr>
+                        <td>6:00pm</td>
+                        <td>Opening Ceremony</td>
+                        <td>CUC McConomy</td>
+                    </tr>
+                    <tr>
+                        <td>7:00pm</td>
+                        <td>Dinner</td>
+                        <td>NSH Atrium</td>
+                    </tr>
+                    <tr>
+                        <td>7:30pm</td>
+                        <td>Hacking begins!</td>
+                        <td>Hack areas</td>
+                    </tr>
+                    <tr>
+                        <td>12:00am</td>
+                        <td>Midnight snack</td>
+                        <td>Rashid</td>
+                    </tr>
+                </table>
+            </div>
+            <div class="schedule-holder">
+                <h2>Saturday</h2>
+                <table>
+                    <thead>
+                        <tr>
+                            <td>When</td>
+                            <td>What</td>
+                            <td>Where</td>
+                        </tr>
+                    </thead>
+                    <tr>
+                        <td>7:30am</td>
+                        <td>Breakfast</td>
+                        <td>NSH Atrium</td>
+                    </tr>
+                    <tr>
+                        <td>12:00pm</td>
+                        <td>Lunch</td>
+                        <td>NSH Atrium</td>
+                    </tr>
+                    <tr>
+                        <td>5:00pm</td>
+                        <td>Dinner</td>
+                        <td>NSH Atrium</td>
+                    </tr>
+                    <tr>
+                        <td>6:00pm</td>
+                        <td>Hacking ends!</td>
+                        <td>Hack areas</td>
+                    </tr>
+                    <tr>
+                        <td>6:30pm</td>
+                        <td>Hack Expo</td>
+                        <td>CUC Rangos</td>
+                    </tr>
+                    <tr>
+                        <td>8:00pm</td>
+                        <td>Closing Ceremony</td>
+                        <td>CUC McConomy</td>
+                    </tr>
+                </table>
+            </div>
+        </div>
+        <footer>
+            <p>TartanHacks is brought to you by Scottylabs.  Found a bug?  Email us at <a href="mailto:help@scottylabs.org">help@scottylabs.org</a>.</p>
+        </footer>
+        <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
```

Comment:
Let's slap some Google Analytics on here. I've just created an analytics code under the cmu.scottylabs@gmail.com account. The access code is UA-54783750-2 or you can just slap this on:

```javascript
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-54783750-2', 'auto');
  ga('send', 'pageview');

</script>
```